Source/gg2/Scripts/Killtable/killtable_delete.gml

```diff
@@ -0,0 +1,8 @@
+//Removes a player and its corresponding value
+//Arg0: Killtable
+//Arg1: Player
+player_list = ds_list_find_value(argument0, 0);
+value_list = ds_list_find_value(argument0, 1);
+index = ds_list_find_index(player_list, argument1);

```

Comment:
Just as feedback (again, I'll fix it myself): You should use local variables (var keyword) for anything that is only needed within one script, otherwise you store it in whatever instance is currently selected, possibly overwriting variables that are already there (especially nasty in one place where you have a variable called "player" since we use that name all over the place for instance variables). This is something I noticed all over the code.